### ISSUE TITLE
v9.53 is dinged by MS Defender.

### ISSUE THREAD

**Issue by Wizardling** [icechat/IceChat#31]
![Screenshot 2022-02-04 094436](https://user-images.githubusercontent.com/3980145/152425961-d5515841-2b91-46e8-a0a8-e269d21c66d6.png)

**Comment by icechat** [comment]
@Wizardling Oh, that will be a fun one.

**Comment by Wizardling** [comment]
I thought you'd 'enjoy' it ;-) Sorry to be a downer right after a successful bugfix, heh.

**Comment by icechat** [comment]
No problem, I never had this issue before, but I will create a new installer that is signed.

**Comment by icechat** [comment]
@Wizardling I have uploaded a new icechat-setup.exe for IceChat 9.53, could you see if this build is still blocked.

**Comment by Wizardling** [comment]
I'm sorry, it is still flagged by Defender.

**Comment by icechat** [comment]
Is it the same screen  ?  It is now digitally signed.

**Comment by Wizardling** [comment]
Same screen. https://github.com/icechat/IceChat/releases/download/9.53F/icechat-setup.exe - yeah?

**Comment by icechat** [comment]
Yeah, just a newer version of the file. Yeah, my windows defender is not catching it.

**Comment by icechat** [comment]
Do you see it has a Digital Signature on the file now, when you right click it and go to properties:?

**Comment by icechat** [comment]
Hold on, I see the problem now, if I actually looked at that tab myself, it shows it is not trusted.
Back to the drawing board!

**Comment by Wizardling** [comment]
I was just about to say that :-) Good luck.

**Comment by icechat** [comment]
So my self signed certificate is not sufficient here, that seems to be the problem.

**Comment by icechat** [comment]
Ok, so it is confirmed that my self signed certificate is invalid, so I can not do much more then that. I will have to see about getting a valid certificate , if it works.

**Comment by Wizardling** [comment]
Have you considered one of several free or low cost certificate authorities, assuming you're trying to avoid unnecessary expense? Of course the downside of the free ones is they tend to be limited to three months a a time.

**Comment by icechat** [comment]
If I could make one work, lol. I need a specific type of file (PFX) that they don't seem to provide.

**Comment by Wizardling** [comment]
Ah. I'm afraid I lack experience in these matters. My only coding nowadays is writing scripts to automate tasks for myself.

**Comment by icechat** [comment]
I will see if I can find a solution but I am not too worried about it in the mean time,
